tests/unit/gui/lazyfilelist/file_list_defs.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+//#include "ffconf.h"
+#define _MAX_LFN 95
+
+/*
+ * Using limits from marlin:
+ * FILENAME_LENGTH				13
+ * LONG_FILENAME_LENGTH			13 * 5 + 1
+ * MAX_DIR_DEPTH				10
+ * MAXDIRNAMELENGTH   			8
+ * MAXPATHNAMELENGTH			(1 + (MAXDIRNAMELENGTH + 1) * (MAX_DIR_DEPTH) + 1 + FILENAME_LENGTH)
+ * */
+
+#ifndef MAX_DIR_DEPTH
+    #define MAX_DIR_DEPTH 10 // Maximum folder depth
+#endif
+
+#ifndef MAXDIRNAMELENGTH
+    #define F_MAXDIRNAMELENGTH _MAX_LFN + 1
+#endif
+
+#ifndef MAXPATHNAMELENGTH
+    #define F_MAXPATHNAMELENGTH (1 + (F_MAXDIRNAMELENGTH + 1) * (MAX_DIR_DEPTH) + 1 + _MAX_LFN)
+#endif
+
+#define strlcpy strncpy
```

Comment:
if we can solve the non-standard strlcpy with clang, this can be removed then

Reply:
No clang for now. I decided to go back to GCC 🙈